Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+members = [
+  "rclrs",
+  "examples/logging_demo",
+  "examples/message_demo",
+  "examples/minimal_client_service",
+  "examples/minimal_pub_sub",
+  "examples/parameter_demo",
+  "examples/rust_pubsub",
+  "examples/worker_demo",
+]
+resolver = "2"
```

Comment:
I'll throw it there, but depending on how long the upstream PR gets to be reviewed, would it make sense to split examples into another repo? This also aligns with how other libraries do it, i.e. [rclcpp](https://github.com/ros2/rclcpp/) only has the library code while the examples are in [examples](https://github.com/ros2/examples/)

Reply:
Yeah, I agree, we're not releasing the examples packages and it'd align with other ROS bindings. I really want to use release-plz to simplify the release process, and given that colcon-cargo has issues with non-Cargo packages in a Cargo workspace https://github.com/colcon/colcon-cargo/issues/63, just moving the examples can make the whole thing easier

Reply:
@luca-della-vedova I just tested moving the examples to another repo and with a `Cargo.toml` workspace file in this repo for `rclrs` and `colcon` can build the whole thing just fine. I'll create a new `examples` repo and submit a PR moving the examples there.